web/src/app/components/Game.tsx

```diff
@@ -134,6 +138,14 @@ export default function Game({roomID}: GameProps) {
             {gameState === GameState.Loading &&
                 <Loader className="mt-40" color="white" />
             }
+            {gameState === GameState.RoomFull && (
```

Comment:
can we move this into it's own component? like Error.tsx or Message.tsx or something?

then we can pass a message string as a prop, and use the same component for the "Game in Progress" error

Reply:
then in that case we can call the game state GameState.Error, and pass the message string as a prop depending on the result of the joinRoom event. just will need to make a state variable for the error message